src/Extension/FluentFilteredExtension.php

```diff
@@ -34,6 +34,14 @@ class FluentFilteredExtension extends DataExtension
      */
     const SUFFIX = 'FilteredLocales';
 
+    /**
+     * Allow the filtered locale behaviour to be skipped for draft stage only
+     *
+     * @var bool
+     * @config
+     */
+    private static bool $apply_filtered_locales_to_stage = false;
```

Comment:
Should this be true by default. Otherwise it may be unintuitive if you have a visible (but draft) page that magically can't publish?